README_Instructions.md

```diff
@@ -0,0 +1,32 @@
+# EdgeX-TAF execution in Pipeline
+
+This is created as an intro to understand how EdgeX-TAF will be executed in pipeline
```

Comment:
@bhavikkothari-eaton This readme file seems to appear as created to run only in pipeline, can we put generic term or to include "Pipeline as well as in local environment" ?
Thought?

Reply:
@rajatkumar-eaton Actually I made this script mainly to run in pipeline because while deploying edgex we are using "**--yocto --conn_method=http**" which is suitable for pipeline only. for local environment we need to remove this line. 

I can mention in this as a note in readme file and can make it pipeline as well as local environment